legacy-app/src/app/modules/events/pages/event-details-page/event-details-page.component.html

```diff
@@ -244,12 +282,9 @@ <h3>
               Nobody yet
             </mat-list-item>
             <mat-list-item *ngFor="let user of event.organizers">
-              <a
-                [routerLink]="
-                  permissions.isAdmin() ? '/tenant/users/' + user.id : null
-                "
-                >{{ user.fullName }}</a
-              >
+              <a [routerLink]="'/tenant/users/' + user.id">{{
```

Comment:
This link should only be there for admins I think as it should not work for others as well

Reply:
Will be fixed with the upcoming #126, when user profiles become public

Reply:
Could we leave it at an empty link until then?